### PR TITLE
make the last imported ifcopenshell file object available as ifcfile …

### PR DESCRIPTION
**Pull request by berndhahnebach**
…in python console

see forum post too ... https://forum.freecad.org/viewtopic.php?p=662939#p662939

### CODE REVIEW

**Comment by yorikvanhavre** [comment]
Good idea... However I'm a bit afraid of what unaware users could do with this :sweat_smile: Maybe we could add a comment to it, let me try...

File: ifc_import.py [hunk 1]
```diff
@@ -49,8 +51,10 @@ def insert(filename, docname, strategy=None, shapemode=None, switchwb=None, sile
         return
     stime = time.time()
     document = FreeCAD.getDocument(docname)
-    ifc_tools.create_document(filename, document, shapemode, strategy)
+    prj_obj = ifc_tools.create_document(filename, document, shapemode, strategy)
     document.recompute()
+    if FreeCAD.GuiUp:
+        FreeCADGui.doCommand("ifcfile = FreeCAD.ActiveDocument.{}.Proxy.ifcfile".format(prj_obj.Name))
```
Comment (yorikvanhavre) [inline ifc_import.py]:
```suggestion
        FreeCADGui.doCommand("ifcfile = FreeCAD.ActiveDocument.{}.Proxy.ifcfile #warning: make sure you know what you are doing when using this!".format(prj_obj.Name))
```

Comment (berndhahnebach) [reply ifc_import.py]: I am totally fine with this.

**Comment by yorikvanhavre** [comment]
okay, let's go then :)

**Comment by berndhahnebach** [comment]
very use full for devs and debuging ... https://github.com/yorikvanhavre/FreeCAD-NativeIFC/issues/23#issuecomment-1443704039

**Pull request merged**